recipes/blas/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set blas_pkg = "openblas" %}
+
+{% set version = "1.0.0" %}
+{% set build_number = 0 %}
+
+{% set version_post = (".post" + build_number|string) if build_number|int != 0 else "" %}
+{% set version_full = version + version_post %}
+
+
+package:
+  name: blas
+  version: {{ version_full }}
+
+build:
+  # Do not change the build number here.
+  number: 0
```

Comment:
This feels sort of silly to me, but we now require build numbers for good reasons. Unfortunately, they just are not appropriate here as they will not show up in package filename because we are changing the build string.